tests/ert/unit_tests/config/test_forward_model_data_to_json.py

```diff
@@ -10,7 +10,7 @@
 from ert.config import ErtConfig, ForwardModelStep
 from ert.config.ert_config import (
     _forward_model_step_from_config_file,
-    forward_model_data_to_json,
+    create_forward_model_json,
```

Comment:
This test file is called "test_forward_model_data_to_json", following this PR it should probably be renamed.